test/classes/InsertEditTest.php

```diff
@@ -1052,6 +1052,32 @@ public function testGetHTMLinput(): void
             . ' class="textfield datetimefield" c tabindex="25" id="field_0_3">',
             $result
         );
+
+        // case 4 int
+        $column['pma_type'] = 'int';
+        $column['True_Type'] = 'int';
```

Comment:
The missing `$column['Type']` should be added.

```suggestion
        $column['True_Type'] = 'int';
        $column['Type'] = 'int(11)';
```